<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>internet_game</class>
 <widget class="QWidget" name="internet_game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>0</y>
     <width>250</width>
     <height>800</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_1">
    <widget class="QPushButton" name="Button_prepare">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>480</y>
       <width>140</width>
       <height>75</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/img/btnbg.jpg);</string>
     </property>
     <property name="text">
      <string>准备</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_exit_room">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>600</y>
       <width>140</width>
       <height>75</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/img/btnbg.jpg);</string>
     </property>
     <property name="text">
      <string>退出房间</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ip">
     <property name="geometry">
      <rect>
       <x>-5</x>
       <y>270</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>IP: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_opponent">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_prepare_">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>未准备</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_fd">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>FD: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="btn_exit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>700</y>
       <width>150</width>
       <height>75</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/img/btnbg.jpg);</string>
     </property>
     <property name="text">
      <string>退出游戏</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_back">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>500</y>
       <width>150</width>
       <height>75</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/img/btnbg.jpg);</string>
     </property>
     <property name="text">
      <string>悔棋</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_surrender">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>600</y>
       <width>150</width>
       <height>75</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/img/btnbg.jpg);</string>
     </property>
     <property name="text">
      <string>认输</string>
     </property>
    </widget>
    <widget class="QLabel" name="Label_your_color">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>310</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_send">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_send">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>270</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_msg">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>420</y>
       <width>180</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="LE_recv">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>180</width>
       <height>250</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_black">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{color:white;border-image: url(:/new/prefix1/img/kuro.png);}
</string>
   </property>
   <property name="text">
    <string>先手</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_agree">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>200</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/img/btnbg.jpg);</string>
   </property>
   <property name="text">
    <string>同意</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_refuse">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>200</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/img/btnbg.jpg);</string>
   </property>
   <property name="text">
    <string>拒绝</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_white">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>540</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/new/prefix1/img/shiro.png);}</string>
   </property>
   <property name="text">
    <string>后手</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>600</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color:rgba(255, 0, 0, 1)}</string>
   </property>
   <property name="text">
    <string>房间名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_victory">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>64</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>黑棋胜利</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_anwser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>64</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>ADSFASDFA</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_prepare">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>800</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>64</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color:rgba(255, 0, 0, 0.6)}</string>
   </property>
   <property name="text">
    <string>请准备...</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_wait_answer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>800</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>44</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>等待对方响应请求</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
